doc/how_to_guides/chomp_planner/chomp_planner_tutorial.rst

```diff
@@ -98,31 +98,3 @@ OMPL is a open source library for sampling based / randomized motion planning al
 CHOMP: While most high-dimensional motion planners separate trajectory generation into distinct planning and optimization stages, CHOMP capitalizes on covariant gradient and functional gradient approaches to the optimization stage to design a motion planning algorithm based entirely on trajectory optimization. Given an infeasible naive trajectory, CHOMP reacts to the surrounding environment to quickly pull the trajectory out of collision while simultaneously optimizing dynamic quantities such as joint velocities and accelerations. It rapidly converges to a smooth, collision-free trajectory that can be executed efficiently on the robot. A covariant update rule ensures that CHOMP quickly converges to a locally optimal trajectory.
 
 For scenes containing obstacles, CHOMP often generates paths which do not prefer smooth trajectories by addition of some noise (*ridge_factor*) in the cost function for the dynamic quantities of the robot (like acceleration, velocity). CHOMP is able to avoid obstacles in most cases, but it can fail if it gets stuck in local minima due to a bad initial guess for the trajectory. OMPL can be used to generate collision-free seed trajectories for CHOMP to mitigate this issue.
-
-Using CHOMP as a post-processor for OMPL
```

Comment:
Why is this section removed? In the example I commented above it was the use case for post-processing.

Reply:
I think in the next part of these PRs, CHOMP will be changed from an adapter to an actual chain of planning pipelines.